packages/core/src/components/sections/ProductDetails/ProductDetails.tsx

```diff
@@ -218,6 +226,15 @@ function ProductDetails({
                 )
               }
               refNumber={showRefNumber && productId}
+              {...(apiConfig.reviewsAndRatings && {
+                reviewsAndRating: {
```

Comment:
suggestion: adding the "s" because I think we're using the term "reviews and ratings" everywhere else, right?
```suggestion
                reviewsAndRatings: {
```